### PR TITLE
Graph

### PR DESCRIPTION
**Pull request by bdduong**

### CODE REVIEW

File: src/client/app/index.jsx [hunk 1]
```diff
@@ -25,28 +23,57 @@ class App extends React.Component {
...
+  generateGraphData(count, percents) {
+    let rating = {
+      5: 'Excellent',
+      4: 'Very good',
+      3: 'Average',
+      2: 'Poor',
+      1: 'Terrible'
+    }
+    let info = [];
+    for (let key in count) {
```
Comment (emsando) [inline src/client/app/index.jsx:48]: is count an array or an object?

Comment (bdduong) [reply src/client/app/index.jsx:48]: count is an object

File: src/client/app/index.jsx [hunk 2]
```diff
@@ -25,28 +23,57 @@ class App extends React.Component {
...
+      info.unshift({rating: rating[key], count: count[key], percentage: percents[key], rank: key});
+    }
```
Comment (emsando) [inline src/client/app/index.jsx:50]: This looks risky to me - but I remember James Powers saying that es6 handles for / in on arrays differently now. But on a first read it looks like something that isn't guaranteed to produce the same order every time. If you can prove me wrong please do, I'd appreciate it !!

File: src/client/app/components/Graph.jsx [hunk 3]
```diff
@@ -1,38 +1,16 @@
...
-  }
+const Graph = (props) => {
+  return (
+    <div className={styles.main}>
+      <p className={styles.header}>Traveler rating</p>
+      <ul className={styles.list}>
+        {props.graphInfo.map(rating => <GraphRow data={rating} handleRating={props.handleRating} />)}
+      </ul>
+    </div>
+  )
```
Comment (emsando) [inline src/client/app/components/Graph.jsx:13]: This looks great

**Pull request merged**